### PR TITLE
ERT Update

### PR DESCRIPTION
**Pull request by meemofcourse**
## About The Pull Request
This pull request is focused on ERTs. It does not add any ERT subshuttles. It does the following:
* Removes the /centcom series of outfits (ERTs, dsquaddies, inspectors, and so on) because the system wasn't a subtype of job outfits and was Old. It does not remove their content.
* Adds the outpost_access boolean to ERTs, which determines if ERT members should be given access to the outpost.
* Does some Misc things:
  * Subtype of CM-15 mag with inc ammo.
  * Subtype of CM-5 mag with rubbershot ammo.
  * Changes up the loot in the ramzi commando outfit a bit.
  * Adds a bunch of subtypes of webbings with ammo.
  * Smokebombs no longer have dead memes in their desc.
* Adds and changes a bunch of fucking ERTs:
  * Indies
indie_erts
    * Deathsquad was moved to here. Also, they get a janitor and pizza delivery team, plus an EVA variant of the EMT team.
  * Roumain
srm_erts
    * This was originally the primary focus of this PR. There's two teams to choose from:
      * A hunting party, composed of a Montagne, a Machinist, a Doctor, and some Hunters.
      * A defense militia, composed of a Montange, a Colligne, a Machinist, a Doctor, and some Hunters. The hunters this time have a variety of ouchy hurt guns, notably the Montange with a two-bore.
  * CLIP 
erts_clip
    * CLIP kind of stole the spotlight from Roumain, with a whopping 6 teams to choose from:
      * Standard team, with a Sergeant, Corpsman, Gunner, Engineer, and the common Minutemen. EVA Variant included.
      * An inspector. For your Inspector ERT needs.
      * A BARD Field Team, with some BARD Field Agents, so you can prove that damn poster wrong.
      * A BARD Emergency Team, with Fire, Medical, Xenofauna Specialists and Master Sergeant, so you can prove that damn poster right.
      * A C-MM Military Police team, with a riot variant, so you can keep safe all of the above.
      * A C-MM Media team, so you can take photos of all of the above.
  * Inteq
inteq_erts
    * Inteq has three ERTs to choose from:
      * A run-of-the-mill team, composed of a Vanguard, Artificer, Corpsman, and some Enforcers. EVA variant.
      * A honor guard team, standing in as a military police team and defending any notable figures.
      * A mothership investigator. Inspector ERT.
  * SolGov
erts_solcon
    * I didn't do much here. Inspector has some cooler gear, I suppose. 
  * NGR
syndie_erts
    * The NGR ERT got updated to use their up-to-date gear, of particular note the Marksman and Grenadier.
    * They also get an official for Inspector ERT Needs.
  * Cybersun
    * Cybersun wasn't touched much.
    *  Brand new Hardliner ERT. 
    * Cybersun Representative, Inspector ERT.
  * PGF
erts_gezena
    * The PGF gets a standard team, with a Gunner, Medic, Engineer and Gunnery Sergeant.
    * There is a Federation Observer, which is the inspector ERT.
  * Ramzi
    * Ramzi has a single ERT to choose from. This basic team should not be underestimated; it has a Demolitionist with a rocket launcher, a Medic with combat simulants, a Leader with access to varying types of shotgun ammunition, and Cell Members with explosives to spare. If they know how to use their gear right, they might not die in a Vaquero airlock.
  * Fronties
erts_fronties
    * The Frontiersmen now have four ERTs to choose from: 
      * A basic team, with Illestern Riflemen and an Officer. There's an unarmed variant if you want to equip them yourself.
      * A raiding team, with Riflemen, an Engineer, a Medic, and an Officer. Balanced, probably.
      * Shock troops, with a Sentry, a Flametrooper, Shock Medic, Shock Officer, and SKM Riflemen. Finally, the sentries see some use. 
      * A randomized team. The chances and equipment were reviewed a bit so you have more chances of getting something Fun instead of Nothing.
  * NT
nt_erts
    * Practically every NT ERT has been killed off except a Vigilitas security team, an N+S delivery team, and CentCom's last stand - the inspector ERT.

## Why It's Good For The Game
please play SRM

## Changelog

:cl:
add: A lot of ERTs and ERT remasters, including Fronties, Ramzi, PGF, NGR, Indies and whatnot
del: Most Nanotrasen ERTs have been shelved pending assets
del: Smokebomb randomized descriptions
/:cl:

### CODE REVIEW

File: code/modules/clothing/outfits/ert/frontiersmen_ert.dm [hunk 1]
```diff
@@ -46,11 +57,11 @@
...
-	if(prob(30))
+	if(prob(50))
 		mask = pickweight(list(
-		/obj/item/clothing/mask/gas/frontiersmen = 5,
-		/obj/item/clothing/mask/gas/sechailer/balaclava = 5,
-		/obj/item/clothing/mask/breath = 5,
+		/obj/item/clothing/mask/gas/frontiersmen = 12,
+		/obj/item/clothing/mask/gas/sechailer/balaclava = 10,
+		/obj/item/clothing/mask/breath = 7,
 		/obj/item/clothing/mask/whistle = 3))
```
Comment (rye-rice) [inline code/modules/clothing/outfits/ert/frontiersmen_ert.dm]:
```suggestion
		/obj/item/clothing/mask/whistle/trench = 3))
```

File: code/modules/admin/verbs/one_click_antag.dm [hunk 2]
```diff
@@ -226,6 +226,7 @@
 	.["mainsettings"]["spawn_admin"]["value"] = newtemplate.spawn_admin ? "Yes" : "No"
 	.["mainsettings"]["use_custom_shuttle"]["value"] = newtemplate.use_custom_shuttle ? "Yes" : "No"
 	.["mainsettings"]["spawn_at_outpost"]["value"] = newtemplate.spawn_at_outpost ? "Yes" : "No"
+	.["mainsettings"]["outpost_access"]["value"] = newtemplate.spawn_at_outpost ? "Yes" : "No"
```
Comment (meemofcourse) [inline code/modules/admin/verbs/one_click_antag.dm]:
```suggestion
	.["mainsettings"]["outpost_access"]["value"] = newtemplate.outpost_access ? "Yes" : "No"
```

**Comment by meemofcourse** [comment]
undrafted

**Review by Sun-Soaked — APPROVED** [review]
looks good

**Comment by Sun-Soaked** [comment]
resolve conflicts and I'll merge

**Pull request merged**